app/views/schedules/_fields.html.erb

```diff
@@ -11,6 +11,7 @@
              Task.where(schedule_id: f.object.id).map { |t| ["##{t.id}", t.id] },
              {include_blank: true, selected: (f.object.nil? || f.object.root_task.nil? ? '' : f.object.root_task.id)},
              class: 'form-control' %>
+
```

Comment:
You'll want to remove these extra spaces in the file if you don't do anything else with it as this progresses. No need to include a whitespace-only change.